src/main/java/co/cask/gcp/bigquery/BigQuerySource.java

```diff
@@ -144,35 +100,17 @@ public void prepareRun(BatchSourceContext context) throws Exception {
     AvroBigQueryInputFormat.setTemporaryCloudStorageDirectory(configuration, temporaryGcsPath);
     AvroBigQueryInputFormat.setEnableShardedExport(configuration, false);
     BigQueryConfiguration.configureBigQueryInput(configuration, config.getProject(), config.dataset, config.table);
+
+    JobID jobID = JobID.forName(String.format("job_%s-%s-%s_%s", context.getNamespace(),
```

Comment:
what is this and the next line for? Seems like it gets a temp path but never uses it?

Reply:
yea its not used. Removed.